zappa/handler.py

```diff
@@ -336,9 +336,27 @@ def handler(self, event, context):
 
         # This is a direct command invocation.
         elif event.get('command', None):
+            if not event['command']:
```

Comment:
Line 338 checks for the existence of the key/value of 'command in the dictionary, but doesn't check if that value is a "false" value (None, empty string, etc).